File: cadastroEscolarFront/src/app/models/DTO/alunoTurmaDTO.ts
import { Pessoa } from "../pessoa";
import { Turma } from "../turma";

export class AlunoTurmaDTO{
    id : number;
    aluno : Pessoa;
    turma : Turma

    constructor(id : number, aluno : Pessoa, turma : Turma){
        this.id = id;
        this.aluno = aluno;
        this.turma = turma;
    }
}

File: turma/src/main/java/com/example/Turma/dto/AlunoTurmaDTO.java
package com.example.Turma.dto;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

import com.example.Turma.model.Pessoa;
import com.example.Turma.model.Turma;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class AlunoTurmaDTO implements Serializable {

    private Long id;
    private Pessoa aluno;
    private Turma turma;
}
